bin/crew

```diff
@@ -1853,7 +1853,7 @@ def sysinfo_command(args)
     lsb_release = ENV
   end
 
-  git_commit_message_format = '%h  %s  (%cr)'
+  git_commit_message_format = '%h `%s (%cr)`'
```

Comment:
Why the backticks?

Reply:
It is for markdown styling (\`abc\` shows `abc`)

Reply:
Ummm, I don't think that is going to work like you might think.   Backticks are used to EXECUTE commands.

Reply:
This is not `bash`... In ruby only `#{}` in `""` will be executed (that means the \` does not have any special meaning in strings)

Reply:
Oh I see what you mean😅 The `#{git_commit_message_format}` in the git command is wrapped with `''` so it will not be executed

Reply:
Backticks work in ruby too.  I think you are right about the single quotes.